### ISSUE TITLE
Prebuilt artifacts

### ISSUE THREAD

**Issue by JensRantil** [Expensify/Bedrock#658]
Hey, I was about to give Bedrock a spin ~6 months ago and ran into issues building it on MacOSX, then gave up concluding it was too much work. The issues were unclear and long forgotten by now, but just wanted to create this issue as a feature request to have prebuilt binaries/artifacts. I think it would benefit the project by reducing the friction significantly to start using Bedrock, both for developers, but also for rolling out in production.

The fact that every shop must build Bedrock themselves, host their own binaries somewhere etc. I think it a big friction point to convince ops people it's a good database to go with.

**Comment by quinthar** [comment]
Yep, I totally agree, but this is much easier said than done.  It's a good suggestion, eager for your PRs!

**Comment by JensRantil** [comment]
A good start is probably to focus on creating an official Docker image. It would be isolated and recreatable and avoid issues with dynamic libs etc. Given that Travis CI build is working and there is a script that could be the basis for starting this work.

**Comment by AstraLuma** [comment]
+1 for containers.

Makes it easier to spin up test environments and clusters.

Makes development environments simpler.

Enables Bedrock on container orchestrators.

**Comment by d33p-dev** [comment]
Yep, agree 1000%.  Just spent a few hours chasing down how to build on Ubuntu ARM and on Mac M1....  Mac just failed and no clear path to resolve.  But, at least on Ubuntu I finally figured out I had to installed the build dependencies manually and then reference a comment in the code during a build error that let me know yet another dependency had to be installed to get the build to compile without libpcre errors.

It's currently taking close to an hour to actually build (not sure why it's so slow) and I'll hopefully then be able to actually test it.

I realize there's lots of platforms out there but there must be a way to at least provide a prebuilt binary for Linux x64 and arm64 which would cover a lot of users.
